internal/constants/preprocess/preprocess.go

```diff
@@ -16,7 +16,7 @@ import (
 
 // Constants holds constants that will be preprocessed, meaning the key value parts here will be built into the constants
 // package as actual constants, using the build-time interpretations
-var Constants = map[string]func() string{}
+var Constants = map[string]func() interface{}{}
```

Comment:
nit: I'm not sure making the function return type an `interface{}`  is necessary here as the `OnCI` value you added is still a string

Reply:
Yeah I couldn't get `jen` to actually write it as a boolean :\ But I figure this still gets it a bit closer to accepting mixed types. Can cross the other bridge at a later time if we need to.